spaceship/lib/spaceship/tunes/build_details.rb

```diff
@@ -0,0 +1,48 @@
+module Spaceship
+  module Tunes
+    # Represents the details of a build
+    class BuildDetails < TunesBase
+      # @return (String) The App identifier of this app, provided by iTunes Connect
+      # @example
+      #   "1013943394"
+      attr_accessor :apple_id
+
+      # @return (String) Link to the dSYM file (not always available)
+      #     lol, it's unencrypted http
+      attr_accessor :dsym_url
+
+      # @return [Bool]
+      attr_accessor :include_symbols
+
+      # @return [Integer]
+      attr_accessor :number_of_asset_packs
+
+      # @return [Bool]
+      attr_accessor :contains_odr
```

Comment:
I don't know what this is, how would you describe this value? It's been false for my apps.